lib/Twig/Extension/HelpersExtension.php

```diff
@@ -104,6 +105,21 @@ public function getImageVersionPreview(string $file): string
         return $dataUri;
     }
 
+    /**
+     * @param string $file
+     *
+     * @return string
+     *
+     * @throws \Exception
+     */
+    public function getAssetVersionPreview($file)
+    {
+        $dataUri = 'data:application/pdf;base64,'.base64_encode(file_get_contents($file));
```

Comment:
For PDFs it works fine, but doesn't work for any other file format that would end up as "document" type.

![image](https://user-images.githubusercontent.com/6014195/209674121-ba2a0deb-6860-4d6d-9f9a-a954b748b443.png)

Could you please narrow down by checking if the file is a pdf format or extend the support for other formats and remove this hardcoded `application/pdf` here?

Reply:
@BlackbitDevs 🏓😉 - Thanks!

Reply:
@kingjia90 , @brusch I have added a MIME type instead of the hard-coded `application/pdf`. 
And I have also changed the preview for non-displayable file types: Those versions now get provided as downloads:
<img width="274" alt="Bildschirmfoto 2023-01-09 um 08 44 28" src="https://user-images.githubusercontent.com/8749138/211261008-bda4c35e-8522-4e44-929e-61a8171aa22c.png">